jenkins/gradle-check-assemble.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+IFS=`echo -ne "\n\b"`
+
+docker ps
+docker stop `docker ps -qa` > /dev/null 2>&1
+
+cd search
```

Comment:
Similar question, why is this needed?

Reply:
I have stated many times that search is just a folder name of the repo.
If you just git clone opensearch then the default folder is opensearch not search.
The reason we use search back in the day because opensearch naming is not defined.

If you clone as opensearch (which seems like you do in the docker file), you should remove cd into search and cd into the corresponding folder instead.

Reply:
Sure that makes sense, the question was to @owaiskazi19, do we still need this ?

Reply:
Changed the design. Created a new ubuntu docker image for gradle check purpose and run it in a jenkisnfile. We don't need the scripts anymore.